### PR TITLE
Default moverSecurityContext to true if no configMap

### PR DESCRIPTION
**Pull request by eemcmullan**

### CODE REVIEW

**Review by sseago — APPROVED** [review]

File: controllers/replicationdestination.go [hunk 1]
```diff
@@ -359,6 +359,15 @@ func (r *VolumeSnapshotRestoreReconciler) configureRepDestResticVolOptions(vsr *
 		}
 	}
 
+	// default moverSecurityContext to true
+	if cm == nil {
+		podSC, err := GetPodSecurityContext(vsr.Namespace, vsr.Spec.VolumeSnapshotMoverBackupref.BackedUpPVCData.Name, r.Client)
+		if err != nil {
+			return nil, err
+		}
+		repDestResticVolOptions.MoverSecurityContext = podSC
```
Comment (kaovilai) [inline controllers/replicationdestination.go:368]: Is it possible to put this in a function and reuse with line 352-356 as well?

**Pull request merged**